_articles/how-to-contribute.md

```diff
@@ -31,6 +31,10 @@ Contributing to open source can be a rewarding way to learn, teach, and build ex
 
 Why do people contribute to open source? Plenty of reasons!
 
+### top techs joined open source
+
+microsoft joins open chain project as platinum member,edX started open edX,AWS opensources Neo-AI project to accelerate machine learning deployments on edge services and so on.
```

Comment:
Capitalize the first letter. Maybe this would be better as a bullet point list? like:
Some big companies joined pretty interesting Open Source projects:
* Microsoft joined Open Chain Project as a platinum member
* edX started Open edX
* AWS opensources Neo-AI project to accelerate machine learning deployments on edge services

Reply:
I’m sorry. I see you already fixed the punctuation.